### PR TITLE
Fix top bar updates

### PR DESCRIPTION
**Pull request by i1i1**
This pr fixes stale of the top bar in the ui:
image

As we don't send node updates to the frontend reactively, we need to send updates periodically. In future, we can just send all updates like that.

### CODE REVIEW

File: backend/telemetry_core/src/aggregator/aggregator.rs [hunk 1]
```diff
@@ -92,6 +92,20 @@ impl Aggregator {
                 send_node_data,
             ));
         } else {
+            tokio::task::spawn({
+                let tx_to_aggregator = tx_to_aggregator.clone();
+                let mut timer = tokio::time::interval(std::time::Duration::from_secs(10));
```
Comment (nazar-pc) [inline backend/telemetry_core/src/aggregator/aggregator.rs]: Should should be something like 1 second instead. Block production happens every 6 seconds, time slot is 1 second. Sending every 10 seconds would mean we don't see new block produced for quite a while.

Comment (i1i1) [reply backend/telemetry_core/src/aggregator/aggregator.rs]: I guess we can supply that from cli then :)

**Review by nazar-pc — APPROVED** [review]

**Comment by nazar-pc** [comment]
Though I do think something like `--update-interval` CLI parameter would make more sense and you also need to update `docker-compose.yml` to include it.

**Comment by i1i1** [comment]
> Though I do think something like `--update-interval` CLI parameter would make more sense and you also need to update `docker-compose.yml` to include it.

Hmm, should we just have as close configuration in `docker-compose.yml` as possible to our production server then?

**Comment by nazar-pc** [comment]
> Hmm, should we just have as close configuration in docker-compose.yml as possible to our production server then?

In `subspace-customizations` it is literally the same as in production, that was the point: you pull new changes, build and `docker compose up -d`. Do we run something different from what in the repo now?

**Comment by i1i1** [comment]
Yeah, we don't disable node list on the frontend by default on production, so we apply changes locally on prod machine.

**Comment by nazar-pc** [comment]
I didn't understand what you said there, sounded like two conflicting statements

File: docker-compose.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ services:
       - /var/run:uid=101,gid=101
       - /app/tmp:uid=101,gid=101
     environment:
-      SUBSTRATE_TELEMETRY_URL: wss://telemetry.subspace.network/feed
-      DISABLE_NODE_LIST: 0
+      SUBSTRATE_TELEMETRY_URL: ws://localhost:8000/feed
```
Comment (nazar-pc) [inline docker-compose.yml]: This is not a URL we use in production

Comment (i1i1) [reply docker-compose.yml]: Yeah, sorry. My bad

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**